app/test_main.py

```diff
@@ -1,3 +1,61 @@
 from app.main import get_human_age
+import pytest
 
-# write your code here
+
+class TestPetAgeInHumanYears:
```

Comment:
By the way, you don't need to introduce a class if it's not for test categorization. And since you only have one test... You clearly don't need it if you don't intend to extend the test suite later.
I'm not sure if that's an issue, though.